shell/platform/tizen/flutter_tizen_texture_registrar.h

```diff
@@ -0,0 +1,53 @@
+// Copyright 2020 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
```

Comment:
The copyright info in this file is different from that of the source file.

Reply:
Update  copyright info